protocol/src/protocol.callHierarchy.proposed.md

```diff
@@ -0,0 +1,142 @@
+
+#### Call Hierarchy
+
+The LSP provides retrieving the call hierachy information with the following request.
+
+_Client Capabilities_:
+
+```ts
+CallHierarchyClientCapabilities {
```

Comment:
I propose changing the name to `CallHierarchyCapabilities`. Client capabilities inside the `TextDocumentClientCapabilities` do not have the `Client` segment in their name:
```
SynchronizationCapabilities
CompletionCapabilities
HoverCapabilities
SignatureHelpCapabilities
ReferencesCapabilities
DocumentHighlightCapabilities
...
```

What do you think?

Reply:
Sounds good!

Reply:
Thanks for the prompt reply! I will go with `CallHierarchyCapabilities` in LSP4J.